src/utilities/git.jl

```diff
@@ -89,3 +89,13 @@ function git_get_master_branch(master_branch::DefaultBranch)
     return string(strip(read(`git rev-parse --abbrev-ref HEAD`, String)))
 end
 git_get_master_branch(master_branch::AbstractString) = master_branch
+
+function git_remote_add_and_seturl(remote_name::AbstractString, url::AbstractString)
+    try
```

Comment:
Instead of a try-catch, can we:

1. First, get the list of all existing remotes.
2. Then, check if `remote_name' is in the list of existing remotes.
3. Then, decide which command to run based on the result from step 2.